src/open_inwoner/conf/parts/contactform_registration.html

```diff
@@ -1,29 +1,89 @@
-<p>Beste</p>
+<table role="presentation" border="0" cellspacing="0" cellpadding="0" width="100%" class="table-mail">
+    <tbody>
+    <tr>
+        <td class="td-mail td__padding-all" colspan="6">
+            <p>Beste inwoner,</p>
+            <p>Wij hebben uw vraag ontvangen.</p>
+        </td>
+    </tr>
+    <tr>
+        <td class="spacing__extra" colspan="6">
+            &nbsp;
+        </td>
+    </tr>
+    <tr class="tr--responsive">
+        <td class="td--responsive td-mail td-mail__bg-success td__padding-all td__leftcell" colspan="3">
+            <p><span class="text-color__small-gray-900">Vraag: </span></p>
+            <h2>{{ subject }}</h2>
+        </td>
+        <td class="td--responsive td-mail td-mail__bg-success td__padding-all td__rightcell td__alignright" colspan="3">
+            <p><!-- <span class="text-color__small-gray-900">Vraagnummer: </span><br/>{{ identification }} --></p>
```

Comment:
@Bartvaderkin I commented out the stuff for the question-number/identification for now.

Reply:
Resolving this as it was based on a confusion.